processor/resourceprocessor/README.md

```diff
@@ -2,24 +2,25 @@
 
 Supported pipeline types: metrics, traces
 
-The resource processor can be used to override a resource.
+The resource processor can be used to apply changes on resource attributes.
 Please refer to [config.go](./config.go) for the config spec.
 
-The following configuration options are required:
-- `type`: Resource type to be applied. If specified, this value overrides the
-original resource type. Otherwise, the original resource type is kept.
-- `labels`: Map of key/value pairs that should be added to the resource.
+`attributes` represents actions that can be applied on resource attributes.
+Only one action is supported for now:
+- `upsert`: Map of key/value pairs that should be added to the resource. 
+In case of conflict with an existing key, the value will be overridden.
 
 Examples:
 
 ```yaml
 processors:
   resource:
     type: "host"
-    labels: {
-      "cloud.zone": "zone-1",
-      "k8s.cluster.name": "k8s-cluster",
-      "host.name": "k8s-node",
+    attributes:
```

Comment:
As discussed offline I think we can borrow the attribute processor `actions` implementation and config for this. 
BTW, if need specifically a `rename` we can add it as an action so that we don't have to use upsert+delete pair all the time.

Reply:
Code was extracted and easy to use.